samples/bluetooth/alexa_gadget/src/main.c

```diff
@@ -0,0 +1,356 @@
+/*
+ * Copyright (c) 2020 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+ */
+
+/** @file
+ *  @brief Alexa Gadgets sample
+ */
+
+#include <zephyr.h>
+#include <stdlib.h>
+#include <stdio.h>
+
+#include <bluetooth/bluetooth.h>
+#include <bluetooth/gatt.h>
+#include <dk_buttons_and_leds.h>
+#include <settings/settings.h>
+
+#include <logging/log.h>
+LOG_MODULE_REGISTER(main);
```

Comment:
Is there a technical reason for this, or just convention?
Logger seems to be functionally superior to printk, and it would be more consistent to use logger everywhere instead of printk in some places.

Reply:
Rather, It is a convention that in the samples we use `printk` and logger is used in modules

Reply:
Ok, I can change to use printk in main